docs/sdks/summary-sdks.md

```diff
@@ -26,6 +26,6 @@ The following SDK documentation is available for UID2 integration.
 | :--- | :--- | :--- | :--- |
 | UID2 SDK for JavaScript | [UID2 SDK for JavaScript](client-side-identity.md) | Client-Side JavaScript SDK that facilitates the process of establishing client identity using UID2 and retrieving UID2 tokens for publishers. | Publishers |
 | UID2 SDK for Java | [UID2 SDK for Java (Server-Side) Reference Guide](uid2-sdk-ref-java.md) | An SDK for audiences using Java server-side:<br/>- Helps publishers to generate or refresh UID2 tokens from [DII](../ref-info/glossary-uid.md#gl-dii).<br/>- Helps UID2 sharers to encrypt or decrypt UID2 tokens.<br/>- Helps DSPs to decrypt UID2 tokens from bid requests. | Publishers<br/>DSPs<br/>Advertisers<br/>Data Providers |
-| UID2 SDK for Python | [UID2 SDK for Python (Server-Side) Reference Guide](uid2-sdk-ref-python.md) | An SDK for audiences using Python server-side:<br/>- Helps DSPs to decrypt UID2 tokens from bid requests.<br/>- Helps UID2 sharers to encrypt or decrypt UID2 tokens. | DSPs<br/>Advertisers<br/>Data Providers |
+| UID2 SDK for Python | [UID2 SDK for Python (Server-Side) Reference Guide](uid2-sdk-ref-python.md) | An SDK for audiences using Python server-side:<br/>- Helps publishers to generate or refresh UID2 tokens from [DII](../ref-info/glossary-uid.md#gl-dii).<br/>- Helps UID2 sharers to encrypt or decrypt UID2 tokens.<br/>- Helps DSPs to decrypt UID2 tokens from bid requests. | Publishers<br/>DSPs<br/>Advertisers<br/>Data Providers |
```

Comment:
On this line, we added some content for the new functionality, but also we lost this:

Helps DSPs to decrypt UID2 tokens from bid requests.

Is it no longer applicable? I believe it is. Or maybe it's redundant since we list sharers? Just checking anyway.

Reply:
It is still there but the order of the list changed

Reply:
TY!

Reply:
TY!